apps/app/src/app/pages/(components)/components/(data-table)/data-table.page.ts

```diff
@@ -39,10 +39,7 @@ export const routeMeta: RouteMeta = {
 	],
 	template: `
 		<section spartanMainSection>
-			<spartan-section-intro
-				name="Data Table"
-				lead="Powerful table and datagrids similar to Angular Material Tables.."
-			/>
+			<spartan-section-intro name="Data Table" lead="Powerful table and datagrids similar powered by TanStack Table" />
```

Comment:
Do you want to add an about section for tanstack table with a link to the docs, like we have it on the Sonner page - https://spartan.ng/components/sonner#about